extras/kms/repository_root_key.go

```diff
@@ -155,3 +155,18 @@ func (r *repository) ListRootKeys(ctx context.Context, opt ...Option) ([]*rootKe
 	}
 	return keys, nil
 }
+
+// ScopeRootKey will lookup the rootKey for a given scope id. Supported options: WithReader
+func (r *repository) ScopeRootKey(ctx context.Context, scopeId string, opt ...Option) (*rootKey, error) {
+	const op = "kms.(repository).FindRootKey"
```

Comment:
FindRootKey vs ScopeRootKey

Reply:
`Find` feels like it's something that can return multiple results and LookupRootKey(...) is already taken.  Not the best name but it seems to accurately describe the what's returned and it's not overly verbose

Reply:
Same here, I think @jefferai is calling out the mismatch between function and `op` string.

However, I agree that `Find` implies the potential for multiple results, but I do find this function name opaque. Perhaps something like `GetRootKey`, `RetriveRootKey`, `RootKeyForScope`, or `LookupRootKeyForScope`?

Reply:
Yeah, I iterated through those names as well.   For now, I think I'll keep it the same unless there's something significantly better suggested.  ty!

Reply:
I think the issue with `ScopeRootKey` is that it is ambiguous if `Scope` is a noun or verb. To me given that functions often begin with a verb, `ScopeRootKey` seems like it is "scoping" a root key, so adding or applying a scope to a root key. I think prefer `LookupRootKeyForScope` or `LookupRootKeyByScope` since:

- The use of `ForScpoe`/`ByScope` removes the ambiguity, scope is clearly used as a noun.
- It uses the same naming as the other Lookup method, `LookupRootKey` so it will be naturally grouped with that function in docs.